src/e2e/pages/authPage.ts

```diff
@@ -71,4 +71,34 @@ export class AuthPage {
     expect(verificationCode).toBeDefined();
     await this.enterVerificationCode(verificationCode as string);
   }
+
+  async signInToFxA(email: string, password: string) {
```

Comment:
It's not clear to me (because I'm not terribly familiar with our existing e2e tests) why this extra method is needed - don't we already have methods to sign in?

Reply:
This method is for logging explicitly in to the Monitor Accounts application and not Monitor.